pkg/http/api.go

```diff
@@ -224,6 +233,32 @@ func (a *api) MarkStatusAsOutboundReady() {
 	a.outboundReadyStatus = true
 }
 
+// Workflow Component: Component specified in yaml (temporal, etc..)
+// Workflow Type: Name of the workflow to run (function name)
+// Instance ID: Identifier of the specific run
+func (a *api) constructWorkflowEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{fasthttp.MethodGet},
+			Route:   "workflows/{workflowComponent}/{workflowType}/{instanceId}",
```

Comment:
Is the `workflowComponent` the actual component (i.e `temporal`, `built-in`, etc.) or is it the _name_ of the workflow engine component? (i.e a "temporal workflow component named `foobar`")

Reply:
This is from the proposal document: The {workflowComponent} segment is the name of a workflow component – i.e., something that would be described in a workflow.yaml file (similar to how state store and pub/sub components have names)